### ISSUE TITLE
关于训练过程，请教

### ISSUE THREAD

**Issue by newoneincntk** [Xiaobin-Rong/ul-unas#14]
容博士，您好，拜读您的又一遍佳作，其中有如下描述：
The learning rate increases linearly from 10−6 to 10−3 over the first 25,000 steps and then decays cosine-wise until 250,000 steps, at which point the training procedure ends. 
是说训练到250000步结束了吗？
整个训练集是720000个样本，batchsize是32，每个样本只用了10次吗？
谢谢！

**Comment by Xiaobin-Rong** [comment]
您好，您的理解是对的。训练持续了250,000个 steps，每个 epoch 是 1,250 steps，所以一共持续了200 steps
